examples/step-58/doc/results.dox

```diff
@@ -0,0 +1,207 @@
+a solution
+@f[
+  \psi(\mathbf x, t) = \psi(\mathbf x, 0) e^{-i V t}.
+@f]
+Because $V$ is large, this means that the phase *rotates quite rapidly*.
+If you focus on the semi-transparent outer part of the domain, you can
+see that. If one colors this region in the same way as the inner part of
+the domain, this rapidly flashing outer part may be psychedelic, but is also
+distracting of what's happening on the inside; it's also quite hard to
+actually see the radiating waves that are easy to see at the beginning
+of the video.
+
+
+<a name="extensions"></a>
+<h3>Possibilities for extensions</h3>
+
+<h4> Better linear solvers </h4>
+
+The solver chosen here is just too simple. It is also not efficient.
+What we do is just punt the matrix to a sparse direct solver in every
+time step and let it find the solution of the linear system. But we
+know that we could do far better:
+
+- First, we should make use of the fact that the matrix doesn't
+  actually change from time step to time step. This is an artifact
+  of the fact that we here have constant boundary values and that
+  we don't change the time step size -- two assumptions that might
+  not be true in actual applications. But at least in cases where this
+  does happen to be the case, it would make sense to only factorize
+  the matrix once (i.e., compute $L$ and $U$ factors once) and then
+  use these factors for all following time steps until the matrix
+  $C$ changes and requires a new factorization. The interface of the
+  SparseDirectUMFPACK class allows for this.
+
+- Ultimately, however, sparse direct solvers are only efficient for
+  relatively small problems, say up to a few 100,000 unknowns. Beyond
+  this, one needs iterative solvers such as Conjugate Gradients (for
+  symmetric and positive definite problems) or GMRES. We have used many
+  of these in other tutorial programs. In all cases, they need to be
+  accompanied by good preconditioners. For the current case, one
+  could in principle use GMRES -- a method that does not require
+  any specific properties of the matrix -- but would be better
+  advised to implement an iterative scheme that exploits the one
+  structural feature we know is true for this problem: That the matrix
+  is complex-symmetric (albeit not Hermitian).
+
+
+<h4> Boundary conditions </h4>
+
+In order to be usable for actual, realistic problems, solvers for the
+nonlinear Schr&ouml;dinger equation need to utilize boundary conditions
+that make sense for the problem at hand. We have here restricted ourselves
+to simple Neumann boundary conditions -- but these do not actually make
+sense for the problem. Indeed, the equations are generally posed on an
+infinite domain. But, since we can't compute on infinite domains, we need
+to truncate it somewhere and instead pose boundary conditions that make
+sense for this artificially small domain. The approach widely used is to
+use the Perfectly Matches Layer method that corresponds to a particular
```

Comment:
`Perfectly Matches Layer` -> `Perfectly Matched Layer` ([wiki link for reference)](https://en.wikipedia.org/wiki/Perfectly_matched_layer)?